examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -381,8 +381,9 @@ function saturated_surface_conditions(
 end
 
 function vertical_diffusion_boundary_layer_tendency!(Yₜ, Y, p, t)
+    (; coupled) = p
```

Comment:
I think we may need to move this inside the `bycolumn` loop to reduce allocations when running with threading